src/handlers/transfers/handler.js

```diff
@@ -665,8 +665,25 @@ const fulfil = async (error, messages) => {
         await TransferService.handlePayeeResponse(transferId, payload, action)
         const eventDetail = { functionality: TransferEventType.POSITION, action }
         // Key position fulfil message with payee account id
+        let topicNameOverride
+        if (action === TransferEventAction.COMMIT) {
+          topicNameOverride = Config.KAFKA_CONFIG.EVENT_TYPE_ACTION_TOPIC_MAP?.POSITION?.COMMIT
+        } else if (action === TransferEventAction.RESERVE) {
```

Comment:
I am not sure how we handle RESERVE messages. Let's discuss.

Reply:
Its a simple if condition that replaces the body. To my knowledge the notification handler does the rest.
It's a quick functional fix, but I'll have to add tests and the sort.